web-server/src/utils/auth.ts

```diff
@@ -3,6 +3,8 @@ import { isNil, reject } from 'ramda';
 
 import { Integration } from '@/constants/integrations';
 
+const DEFAULT_GH_URL = 'https://api.github.com';
```

Comment:
can have this defined in a constant.ts file as this is being used in multiple places